### ISSUE TITLE
Tomatoes 🍅 

### ISSUE THREAD

**Issue by nelsonic** [dwyl/health#91]
https://youtu.be/wfsJEGMmycI

**Comment by iteles** [comment]
I haven't watched the video yet, but my only note on tomatoes is that they are [nightshades](https://blog.paleohacks.com/nightshades/#).

Whilst many people are completely unaffected by nightshades, there is a growing number of people with a sensitivity to them and they have also been 'known to aggravate' conditions of inflammation and joint pain.

I couldn't find any scientific papers on this so for now, it is pretty much all anecdotal, but many detox programs and elimination diets now recommend abstaining from nightshades for the duration of the program and adding them back in _slowly_ to determine possible sensitivities.

All this to say, tomatoes have [amazing, antioxidant properties](https://www.drfuhrman.com/library/eat-to-live-blog/107/you-say-tomato-we-say-lycopene-a-protective-carotenoid-to-fight-cancer-heart-disease), but each person should test the effects on their own bodies.

**Comment by nelsonic** [comment]
@iteles agreed. (_each person should be **responsible** for determining the effects on their own health_)
Worth noting that much of the "fresh" tomatoes people eat in Europe are "factory farmed" in greenhouses in Morocco:

https://ec.europa.eu/eurostat/statistics-explained/index.php/The_fruit_and_vegetable_sector_in_the_EU_-_a_statistical_overview
![image](https://user-images.githubusercontent.com/194400/47265529-1bb0aa80-d521-11e8-911e-612bfbd99a10.png)

Further reading:
+ https://www.slideshare.net/greensmile/production-of-greenhouse-tomatoes-in-morocco
+ https://www.moroccoworldnews.com/2018/07/250936/morocco-worlds-largest-tomato-exporter
+ https://www.worldatlas.com/articles/which-are-the-world-s-leading-tomato-producing-countries.html
+ https://www.researchgate.net/publication/225542464_Environmental_impact_of_greenhouse_tomato_production_in_France

Similar story in North America: http://ucce.ucdavis.edu/files/datastore/234-447.pdf 37% greenhouse.

in other words tomatoes are "engineered" with nutrient/fertiliser solutions and sprayed with pesticides

We will be growing Organic Heirloom Tomatoes in the garden and Greenhouse https://github.com/dwyl/home/issues/14
But they won't be for everyone; by _definition_ we won't be _able_ to grow "enough"
for everyone in the space we have available ... 
if each person eats one tomato per day, it's 20 people x 365 days = **7,300**! 😮 
We will ("only") grow a few _hundred_ tomatoes per year so we will still need to buy canned/fresh ones.

Greenhouse Tomatos:  "_Best way to grow is **no dig** with compost on top_"
[![image](https://user-images.githubusercontent.com/194400/47257991-fa9e7a00-d48c-11e8-8c93-ac4011e4aa02.png)](https://youtu.be/54deYPBqeAQ)
https://youtu.be/54deYPBqeAQ

I am _especially_ excited about growing _delicious_ organic cherry tomatoes for salads 
which we pick each day off the vine and have zero "food miles". 

There are more than **3,000 varieties** of heirloom or heritage tomatoes in active cultivation worldwide and more than 15,000 known varieties. 
https://www.heirloomtomatoplants.com/Heirloom%20-Tomato_Plants_Heirloom_Tomato%20_seeds.htm
![image](https://user-images.githubusercontent.com/194400/47265636-050b5300-d523-11e8-8df2-c11e226dfd8d.png)

I can't _wait_ to grow our own and _experience_ the variety of flavours!

**Comment by rub1e** [comment]
I've grown a couple dozen heirloom varieties over the years. You'll have much more to choose from with Portugal's climate, but here are the ones I've particularly enjoyed:

* Black Opal (explosion of flavour)
* Sweet aperitif
* Gardener's delight (so easy to grow)
* Green Zebra (amazing in a caprese salad)

Also, send $10 to http://old-hos.ifas.ufl.edu/kleeweb/newcultivars.html and they'll send you three seed packets of their new cultivars which are full of #science - the cherry ones were decent, but the bigger ones blew my mind when I grew them a couple of years ago

**Comment by iteles** [comment]
@rub1e also shared this earlier today:

> Unbelievably, spraying a dilute solution of aspirin onto your tomato plants (we are talking half a soluble tablet per litre of water) is capable of causing their sugar content to soar one and a half times and boost their Vitamin C content 50%.

http://www.jameswong.co.uk/tomatoes/4588088979

**Comment by nelsonic** [comment]
@iteles remind me to try the Aspirin trick next year. 😉 

This year we got a few kilos of cherry tomatoes.
![image](https://user-images.githubusercontent.com/194400/66463398-e58f7200-ea74-11e9-8e06-eb0fa59f9764.png)
![image](https://user-images.githubusercontent.com/194400/66462947-04d9cf80-ea74-11e9-9170-e106bda93a71.png)
We used them in salads and to make home made vegan pizza:
![image](https://user-images.githubusercontent.com/194400/66463061-3d79a900-ea74-11e9-90cb-68c170352354.png)
![image](https://user-images.githubusercontent.com/194400/66463247-a2350380-ea74-11e9-9147-7a48994d2fcb.png)
![image](https://user-images.githubusercontent.com/194400/66463121-56825a00-ea74-11e9-8807-6ae038c8092f.png)

Going to take this to the next level in 2020 with a wood fire oven: https://github.com/dwyl/home/issues/77#issuecomment-504650399

**Comment by nelsonic** [comment]
Quite keen to _grow_ our own tomatoes again and make pizza sauce from scratch. 
Pizza Sauce Recipe: https://www.christinascucina.com/best-pizza-sauce-recipe-no-cook-authentic-italian-style/
